exporter/go.mod

```diff
@@ -0,0 +1,47 @@
+module go.opentelemetry.io/collector/exporter
+
+go 1.18
+
+require (
+	github.com/cenkalti/backoff/v4 v4.1.3
+	github.com/stretchr/testify v1.8.0
+	go.opencensus.io v0.23.0
+	go.opentelemetry.io/collector v0.0.0-00010101000000-000000000000
+	go.opentelemetry.io/collector/pdata v0.62.1
+	go.opentelemetry.io/otel v1.11.0
+	go.opentelemetry.io/otel/sdk v1.11.0
```

Comment:
Not immediately problematic, but this direct dependency on the OTel Go SDK in an instrumented library that should only use the API is something we recommend avoiding.  In the go-contrib repo we have done this by having a subordinate `test` module that can depend on the SDK tracetest packages and the like.  It presents a good and reliable signal to consumers that an instrumentation library is not going to attempt to modify your SDK configuration when it doesn't require the SDK module at all.

Reply:
`exporter/go.mod` is no longer in this PR, but I see what you're saying. as the exporterhelper is back in the collector module (which depends on the sdk for different reasons), i think it's ok for now, but if the exporterhelper gets split into its own module, we could do what you're suggesting here to avoid taking on a dependency on the sdk for the tests alone.